01_data_exploration.ipynb

```diff
@@ -10,17 +10,17 @@
     "# 1. Data Exploration\n",
```

Comment:
<p>It might be nice to link to further reading on the Met Office Machine Learning best practices, or some other accessible resource around these important topics. Some could be found in this <a href="https://docs.google.com/document/d/1LuHU8KcvleH7RkRitQux5LtxfZkA2SRxfNnTt6AqSsY/" target="_blank">annotated reading resource produced by a recent workshop on reproducibility</a>, in particular I thought this resource on ML Pitfalls was very accessible and concise: <a href="https://arxiv.org/abs/2108.02497" target="_blank">How to avoid machine learning pitfalls: a guide for academic researchers, Michael A. Lones</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/ml_weather_tutorial/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>In this case, I actually want them to use the documents that we (the met office) have produced. But I cant put the link into the document, since that's on sharepoint and not accesible generally. I've added a sentence reminding people to look at the document on sharepoint.</p>